FirebaseTestProject/Views/ProfileView.swift

```diff
@@ -37,21 +56,38 @@ struct ProfileView: View {
                     .background(Color.blue)
                     .cornerRadius(15)
             }
-            .disabled(signOutTask.isInProgress)
+            .disabled(task.isInProgress)
             Spacer()
         }
-        .alert("Cannot Sign Out", isPresented: $signOutTask.isError, presenting: signOutTask.error, actions: { _ in }) { error in
+        .alert("Error", isPresented: $task.isError, presenting: task.error, actions: { _ in }) { error in
             Text(error.localizedDescription)
         }
+        .alert("Choose Profile Photo", isPresented: $showChooseImageSource) { // When the .confirmationDialog() modifier was used instead of an alert, the other alerts became inoperable (probably a SwiftUI bug), meaning that any potential errors were not propagated to the user.
```

Comment:
Could these not be combined?  Might have missed a note on it.

Reply:
Hard to combine this with an error alert, but I wanted this to be a `confirmationDialog` regardless. I'll go ahead and make that change.